challenges/objects-arrays.js

```diff
@@ -26,27 +47,83 @@ console.log(tyrannosaurus.period);
 // Create a new roar method for the tyrannosaurus.  When called, return "RAWERSRARARWERSARARARRRR!" Log the result.
 console.log(tyrannosaurus.roar());
```

Comment:
This is where you would declare the roar method outside of the tyrannosaurus object.